### ISSUE TITLE
CT: Search results breadcrumb missing 

### ISSUE THREAD

**Issue by sonntag-adam** [department-of-veterans-affairs/va.gov-team#7528]
This bug is able to be reproduced in a similar way it occurs in https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/8228

Steps to reproduce:
**Open browser in incognito mode**

1. Copy and paste https://staging.va.gov/gi-bill-comparison-tool, **don't** select the link
2. For "Which GI Bill benefit do you want to use? (Learn more)" select Montgomery GI Bill
3. Search on College of Charleston and select school on the search results page
4. Observe that search results breadcrumb is missing.

![Screen Shot 2020-04-02 at 9.54.05 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/7ae7d315-6f9b-40f6-87b9-2f483b1ab873)
##
![Screen Shot 2020-04-02 at 9.54.19 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/d33f031e-6c0d-4c0e-ad14-c8ca1db428af)

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/12106

**Comment by ccruzgra** [comment]
@sonntag-adam Noticed this on mobile displays, too. The breadcrumb would normally lead back to the search results from the profile page.  

![Screen Shot 2020-04-03 at 8.46.19 AM.png](https://images.zenhubusercontent.com/5b55212f6dabc3393a2dbe50/d7e805f3-2a31-4e1a-9376-0c4c306d125f)

**Comment by sonntag-adam** [comment]
![Screen Shot 2020-04-03 at 9.57.37 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/58673df7-58ed-4ee8-a929-e7a92e38e0da)
##
![Screen Shot 2020-04-03 at 10.00.55 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/7cc575c3-53b0-474b-9f82-74973e680529)
##
![Screen Shot 2020-04-03 at 10.02.37 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/5ca32be5-2dc2-4305-8bef-5b9e94434e4c)
##
![Screen Shot 2020-04-03 at 10.03.20 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/591a3b2c-38a1-4b7f-8c1c-5df4774a995b)

**Comment by sonntag-adam** [comment]
In staging, closing bug.

**Comment by sonntag-adam** [comment]
Closing as unable to reproduce, will open and add more details in the future if it occurs again.

**Comment by sonntag-adam** [comment]
Updated steps to reproduce

**Comment by sonntag-adam** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/12609

**Comment by dmccurdybah** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/12630

**Comment by sonntag-adam** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/12647

**Comment by sonntag-adam** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/12649

**Comment by sonntag-adam** [comment]
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/48804582/81971165-a880f480-95ee-11ea-9394-5d40582fb915.gif)

**Comment by sonntag-adam** [comment]
Still occurs in staging.

**Comment by dmccurdybah** [comment]
Another PR: https://github.com/department-of-veterans-affairs/vets-website/pull/12889

**Comment by sonntag-adam** [comment]
Testing passes locally

**Comment by sonntag-adam** [comment]
Still occurs in staging.

**Comment by dmccurdybah** [comment]
Additional PR to remove unnecessary react-router `with-router` usage:

https://github.com/department-of-veterans-affairs/vets-website/pull/13096

**Comment by sonntag-adam** [comment]
Still occurs in staging

**Comment by desireeturner** [comment]
Waiting on VSP team to make an update to the react-router library sometime soon and we can see if that fixes this.

**Comment by dmccurdybah** [comment]
PR for react-router-dom update: https://github.com/department-of-veterans-affairs/vets-website/pull/14143

**Comment by sonntag-adam** [comment]
Still occurs in staging.

**Comment by dmccurdybah** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/15023

**Comment by dmccurdybah** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/15052

**Comment by sonntag-adam** [comment]
This still occurs in staging.

**Comment by dmccurdybah** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/15093

**Comment by dmccurdybah** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/15152

**Comment by delli-gatti-michael** [comment]
The fix is in Staging. Closing this out
